Differentiate ver-1c TMAP4 and TMAP7 (#223)
idaholab/TMAP8: doc/content/verification_and_validation/ver-1c.md

loading of 1 atom/m$^3$ in the first 10 m of a 100 m slab. TMAP4 uses a slab length of 2275 m (the slab length is not specified in the TMAP7 document [!cite](ambrosek2008verification)), however using a smaller slab length was found to not change the results. Additionally, the smaller domain size allows getting a finer simulation mesh for the same computational cost, which improves the agreement between the TMAP8 and analytical calculations.

Diffusivity is set to 1 m$^2$/s

> and no trapping is included. The boundary condition on the left-hand side of the slab (at $x=0 \mathrm{\:m}$) is different for the TMAP4 and TMAP7 cases. For TMAP4, an insulating boundary condition is assumed, and the analytical solution is given by:

Reviewer: ```suggestion
and no trapping is included. The boundary condition on the left-hand side of the slab (at $x=0$ m) is different for the TMAP4 and TMAP7 cases. For TMAP4, an insulating boundary condition is assumed, and the analytical solution is given by [!citep](Carslaw1959conduction):
```

\begin{equation}
\label{eq:c_func_4}
C = \frac{C_0}{2} \left[ \text{erf}\bigg(\frac{h-x}{2\sqrt{Dt}}\bigg) +
\text{erf}\bigg(\frac{h+x}{2\sqrt{Dt}}\bigg)  \right]
\end{equation}

> but for TMAP7, which specifies $C(x=0 \mathrm{\:m})=0$, the analytical solution is

Reviewer: ```suggestion
but for TMAP7, which specifies $C(x=0 \mathrm{\:m})=0$, the analytical solution is [!citep](Carslaw1959conduction)
```

\begin{equation}
\label{eq:c_func_7}
C = \frac{C_0}{2}\left[2\mathrm{erf}\left(\frac{x}{2\sqrt{Dt}}\right) - \mathrm{erf}\left(\frac{x-h}{2\sqrt{Dt}}\right) - \mathrm{erf}\left(\frac{x+h}{2\sqrt{Dt}}\right)\right]
\end{equation}

where $h=10$ m is the thickness of the pre-loaded portion of the layer.

!alert warning title=Typo in [!cite](longhurst1992verification)

> The value of $C$ found in [!cite](longhurst1992verification) has a typographical error, $\sqrt{Dt}$ should be at the denominator. [eq:c_func_4] follows the form of [!cite](Carslaw1959conductionadiabatic).

Reviewer: ```suggestion
The value of $C$ found in [!cite](longhurst1992verification) has a typographical error, $\sqrt{Dt}$ should be at the denominator. [eq:c_func_4] follows the form of [!cite](Carslaw1959conduction).
```

> TMAP4 and TMAP7 verification cases are also evaluated at slightly different locations: TMAP4 verifies the mobile species concentration at three points - 

Reviewer: ```suggestion
TMAP4 and TMAP7 verification cases are also evaluated at slightly different locations: TMAP4 verifies the mobile species concentration at three points:
```

## Input files

!style halign=left
The input file for this case can be found at [/ver-1c.i], which is also used as test in TMAP8 at [/ver-1c/tests]. The TMAP4 and TMAP7 verification tests use the same input file, 

> but different command line arguments for TMAP4

Reviewer: ```suggestion
but different command line arguments for TMAP4.
```
